force-app/main/default/lwc/serviceDeliveryRow/serviceDeliveryRow.js

```diff
@@ -531,7 +530,7 @@ export default class ServiceDeliveryRow extends LightningElement {
                 }
             });
 
-            if (this.hasContactField && contactId) {
+            if (this.hasContactField && this.hasProgramEngagementField && contactId) {
```

Comment:
@bmuniswamy - do we not need to set selected contact for anything else besides handleGetSserviceEngagements?

Reply:
No @jjbennett , I don't think so